src/psyclone/tests/domain/lfric/lfric_integer_builtins_test.py

```diff
@@ -125,6 +125,15 @@ def test_int_X_plus_Y(tmpdir, monkeypatch, annexed, dist_mem):
             "f2_proxy%data(df)\n"
             "      END DO")
         assert output in code
+
+        # Test the lower_to_language_level() method
```

Comment:
This isn't needed because the Integer builtins don't have their own `lower_to_language_level()` methods so this is effectively just testing the non-integer version which of course we already do elsewhere.

I think the same is true throughout this file?

Reply:
@arporter, I followed your example of testing for `int_a_minus_X` where there is also a test for `lower_to_language` method. Besides, isn't all the code generation ultimately going to be done trhough PSyIR? If so, then we will be expanding the testing of lowering and removing the current testing of `gen_code`. Therefore I would like to leave those tests in there.

Reply:
Oops, that's my mistake then - there's no need to test a routine that doesn't exist!

Reply:
OK. Would you like me to remove all `lower_to_language_level()` tests for integer built-ins, including yours? I think leaving the tests for the code generation that covers declarations is essential as this is where the real difference is between the `real` and `integer`-valued built-ins (different field types).

Reply:
Yes please and yes, I agree - we definitely need the existing code-generation tests.

Reply:
Done.